mobify/astro-scaffold
# Forget SplashScreenPlugin existed and use Launch Screens instead

Exactly what the title says

JIRA: https://mobify.atlassian.net/browse/HYB-889
Linked PRs: https://github.com/mobify/astro/pull/590
## Changes
- Add a launch screen for Android
## How to test-drive this PR
- Run the app
- Observe launch screen
## TODOS:
